src/v/pandaproxy/handlers.cc

```diff
@@ -185,17 +192,23 @@ create_consumer(server::request_t rq, server::reply_t rp) {
           parse::error_code::invalid_param, "auto.commit must be false");
     }
 
-    return rq.ctx.client.create_consumer(group_id, req_data.name)
-      .then([group_id, res_fmt, rq{std::move(rq)}, rp{std::move(rp)}](
-              kafka::member_id name) mutable {
+    return rq.ctx.client.invoke_on(
```

Comment:
you need some `smp` accounting here, see the raft dispatch for x-shard accounting